dotcom-rendering/src/web/components/ContainerTitle.tsx

```diff
@@ -35,7 +35,8 @@ const headerStyles = (fontColour?: string) => css`
 	${headline.xsmall({ fontWeight: 'bold' })};
 	color: ${fontColour ?? neutral[7]};
 	padding-bottom: ${space[1]}px;
-	padding-top: ${space[1]}px;
+	padding-top: 6px;
```

Comment:
I’m not sure why we would move away from `@guardian/source-foundation`’s space, here

Reply:
Thought I'd commented on that before - comparing the padding-top value that's currently on frontend to the one that gets generated with source, the available source values don't seem to match. space[1] I believe is 4, space[2] is 8.